### ISSUE TITLE
antew/vim-elm-language-server merged into w0rp/ale

### ISSUE THREAD

**Issue by andys8** [cappyzawa/medium#1]
Regarding https://github.com/cappyzawa/medium/blob/master/docs/20190605.md

Once https://github.com/w0rp/ale/issues/2592 is merged and the plugin updated, `antew/vim-elm-language-server` can be replaced. The linter name is `elm_ls`. See https://github.com/elm-tooling/elm-language-server/pull/80/files for changes.

Elm language server can also be installed with `npm i -g @elm-tooling/elm-language-server` because the package was published :)

I created https://github.com/elm-tooling/elm-vim and I like your article. Maybe there is something you can propose to add.

**Comment by cappyzawa** [comment]
@andys8 Hi.
Thank you for reading despite the Japanese article!
And Thanks for your information.

I will add your propose to my article.

**Comment by cappyzawa** [comment]
TODO:
- [ ] add `npm i -g @elm-tooling/elm-language-server`
- [ ] watching w0rp/ale#2592
